### DISCUSSION TITLE
Linux Mint 22 vs MPD dependencies

### DISCUSSION THREAD

**Discussion by johnmc98683** [MusicPlayerDaemon/MPD#2103] [General]
During the upgrade of Mint 21.3 to Mint 22 a number of updates were installed. One of these was libfmt9, from libfmt8. Unfortunately, mpd-0.23.15, requires libfmt8, as do the previous versions of MPD.  MPD cannot even start due to this error.  Has anyone else seen this problem? Is there an easy solution to this problem?

**Comment by MaxKellermann** [comment]
This problem has nothing to do with this project. It is a bug in your distribution. You should consult Mint support to get this solved.

**Reply by johnmc98683** [reply]
Can I assume from the response that no one else has reported this difficulty? If that's the case, I can further assume that the problem could be in my system, or in the upgrade to Mint. Thank you for your help.
BTW, I really like MPD/MPC.

**Reply by MaxKellermann** [reply]
I've never heard of such a report, but then again why would anybody report Mint bugs here?

**Comment by leigh123linux** [comment]
Linuxmint uses the Ubuntu repo to provide most packages including mpd.
